src/BenchmarkManager.py

```diff
@@ -266,18 +266,66 @@ def load_config(self, config: dict, app_modules: list) -> None:
                         "devices"][single_device] = device_wrapper
 
     @staticmethod
-    def _query_for_config(config: dict, prefix: str = "") -> dict:
-        for key, config_answer in config.items():
+    def _query_for_config(param_opts: dict, prefix: str = "") -> dict:
+        config = {}
+        for key, config_answer in param_opts.items():
+            if not config_answer.get("if") is None:
```

Comment:
I think this can be:
```suggestion
            if config_answer.get("if"):
```